docs/user-management/index.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 1
+title: User Management
+---
+
+This section explains how to view users and interpret the provided information. Refer to [Authentication](../authentication.md) for information about accessing your Macrometa account.
+
+## View Users
+
+Once you log in to a Macrometa account, you can view all users if you have read permissions for the GeoFabric.
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Account**.
+
+Macrometa displays the **Users** tab which lists all user accounts. Type a query in the **Filter users** field to narrow the search and find a user by their ID, email address, tenant, or username.
```

Comment:
I suggest putting filtering as an optional third step.

Reply:
@diana-macrometa Did we ever make a decision on how we want to format optional steps?  For example:

`1. (Optional) Do the thing.`

`1. Optionally, do the thing.`

`1. Do the thing (optional).`

Reply:
The first.

Reply:
Just added it to the style guide.